pkg/maps/nat/stats/stats.go

```diff
@@ -41,8 +43,23 @@ type Stats struct {
 	config   Config
 	natMap4  nat.NatMap4
 	natMap6  nat.NatMap6
+
+	Observable4 stream.Observable[TupleCountIterator]
```

Comment:
I'd expose a method that returns `observable4` instead of exposing the field directly. This makes it easier to locate subscribers by searching for the method's callers.